chia/cmds/wallet_funcs.py

```diff
@@ -44,7 +44,8 @@ async def get_transaction(args: dict, wallet_client: WalletRpcClient, fingerprin
 
 async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
     wallet_id = args["id"]
-    txs: List[TransactionRecord] = await wallet_client.get_transactions(wallet_id)
+    # 1 trillion is over 1500 years of transactions at 20/second, so probably "all"
+    txs: List[TransactionRecord] = await wallet_client.get_transactions(wallet_id, start=0, end=1_000_000_000_000)
```

Comment:
I find this just somewhat unsettling. Maybe we want something more like all=True, or alternative, if start/end are not in the request, we call the existing `get_all_transactions_for_wallet`. I might consider changing the code in wallet_rpc_api.py `get_transactions`. If start/end are not in the request use `get_all_transactions_for_wallet`.

Reply:
I share the distaste for the just-make-it-a-big-number approach.  I figured not changing the RPC interface would be easier to get approved.  It sounds like a more intrusive and interface changing approach is preferred.  Should I take a pass at that?

I'm not a 'web' dev but with a 'regular' API you would expect to request a page of results and then another page.  But, the API has to support a cursor system or some such to handle this without hazarding a transaction being added between two page requests causing the pages to overlap.  So I guess for now a single giant request of some form is going to be the way.

Thanks for the discussion here.

Reply:
One potential issue to my suggestion is that the UI also calls this RPC and it's likely not prepared to receive thousands of transactions by default. So we shouldn't change the default RPC behaviour. However, I think adding a parameter to the request is fine. I would prefer a simplified SQL query and using the existing function to retrieve all the txs - we already have this function, so let's call it.

Reply:
`start` and `end` can already have 'contradictory' values such as `start=5, end=4`.  Maybe it would be good to avoid adding another potentially contradictory parameter for `all`?  The alternative being to make a new 'get all transactions' endpoint.  I forget if we discussed that elsewhere and had a reason not to do that.